src/werkzeug/_reloader.py

```diff
@@ -312,17 +312,29 @@ def run_step(self) -> None:
 
 class WatchdogReloaderLoop(ReloaderLoop):
     def __init__(self, *args: t.Any, **kwargs: t.Any) -> None:
-        from watchdog.events import EVENT_TYPE_OPENED
+        from watchdog.events import EVENT_TYPE_CLOSED
+        from watchdog.events import EVENT_TYPE_CREATED
+        from watchdog.events import EVENT_TYPE_DELETED
+        from watchdog.events import EVENT_TYPE_MODIFIED
+        from watchdog.events import EVENT_TYPE_MOVED
         from watchdog.events import FileModifiedEvent
         from watchdog.events import PatternMatchingEventHandler
         from watchdog.observers import Observer
 
+        event_allowlist = [
```

Comment:
What about calling this `reload_events` or something like that?

(yes, I don't like this `allowlist` newspeak terminology, and I think here we can come up with something better)

Reply:
Sure. I don't mind.